### ISSUE TITLE
quiz next page button not working

### ISSUE THREAD

**Issue by martinvivek** [moodlerooms/moodle-theme_snap#57]
quiz next page button not working. and navgation bar appears with profile menu

**Comment by martinvivek** [comment]
its not working for first question. remaining questions its working fine
using latest 3.1 version of moodle and theme

**Comment by davidscotson** [comment]
Thanks for the report, we're currently working on this. To confirm, are you seeing this when using the secure quiz mode?

**Comment by martinvivek** [comment]
No as a administrator only, i will check once again and confirm

**Comment by martinvivek** [comment]
Yes in the secure quiz mode and as well as normal view and the custom navigation menu appears at the top also

**Comment by davidscotson** [comment]
We think this is fixed, can anyone confirm or deny that?
